### PR TITLE
Add instructions on how to add/remove child elements with  a parent element  #1500

### PR DESCRIPTION
**Pull request by alhambrav**
### Ticket reference or full description of what's in the PR
Add instructions on how to add/remove child elements with  a parent element  
https://github.com/craftercms/craftercms/issues/1500

Update images/text

### CODE REVIEW

**Review by rart — COMMENTED** [review]
Most of this looks like the removal of RTE2. I don't think that you're doing that on 3.1.x and this seems to be going to master.

Was there anything specific I was meant to look into here?

**Comment by alhambrav** [comment]
I guess this part in the file `rte5-setup.rst` beginning on line 288, `Adding support for valid child elements within a parent element` is new in the docs to resolve the issue  `RTE does not support valid HTML5 nested anchor`

Thank you!

File: source/site-administrators/studio/rte5-setup.rst [hunk 1]
```diff
@@ -48,12 +48,21 @@ Let's take a look at an example of using one of the TinyMCE plugins.
 
 The default editor instance contains a menubar with most of the commonly used editing tools.  Sometimes, you want handy buttons available so you don't have to find the tool you need from the menubar.  We'll add a media button to our editor instance to be able to embed a YouTube video:
 
-1. Add the button name **media** to one of the toolbarItem tags: ``<toolbarItems1>``.  An **Insert/Edit Embedded Media** button |rteMediaBtn| will now be available for users of the RTE.
+1. Open the RTE (TinyMCE 5) configuration file in Studio by opening the **Sidebar**, then click on |siteConfig| -> *Configuration* -> *RTE (TinyMCE 5) Configuration*
+2. Add the button name **media** to one of the toolbarItem tags: ``<toolbarItems1>``.  An **Insert/Edit Embedded Media** button |rteMediaBtn| will now be available for users of the RTE.
```
Comment (rart) [inline source/site-administrators/studio/rte5-setup.rst:52]: FYI
- On TinyMCE, buttons can be added through [`toolbar(n)`](https://www.tiny.cloud/docs/configure/editor-appearance/#toolbarn) or [`toolbar`](https://www.tiny.cloud/docs/configure/editor-appearance/#toolbar) but the rte config only supports toolbarItems(n). The new rte config — starting on v4 — will allow almost any tinymce config.
- Tiny's `toolbar(n)` are available only up to 4 through our `toolbarItems{1,2,3,4}`

File: source/site-administrators/studio/rte5-setup.rst [hunk 2]
```diff
@@ -48,12 +48,21 @@ Let's take a look at an example of using one of the TinyMCE plugins.
 
 The default editor instance contains a menubar with most of the commonly used editing tools.  Sometimes, you want handy buttons available so you don't have to find the tool you need from the menubar.  We'll add a media button to our editor instance to be able to embed a YouTube video:
 
-1. Add the button name **media** to one of the toolbarItem tags: ``<toolbarItems1>``.  An **Insert/Edit Embedded Media** button |rteMediaBtn| will now be available for users of the RTE.
+1. Open the RTE (TinyMCE 5) configuration file in Studio by opening the **Sidebar**, then click on |siteConfig| -> *Configuration* -> *RTE (TinyMCE 5) Configuration*
```
[the remark this answers is not in the document]
Comment (alhambrav) [reply source/site-administrators/studio/rte5-setup.rst:51]: Ticket https://github.com/craftercms/craftercms/issues/4857 covers removing the old RTE in the docs,  we can then do the renaming, etc. in the docs through that ticket.

File: source/site-administrators/studio/rte5-setup.rst (hunk 1 above)
Comment (sumerjabri) [reply source/site-administrators/studio/rte5-setup.rst:52]: @alhambrav should we add some of Roy's notes to the docs?

**Review by rart — APPROVED** [review]

**Pull request merged**